tensorrt_llm/_torch/pyexecutor/model_engine.py

```
@@ -2587,10 +2602,22 @@ def _prepare_tp_inputs(
                                         "mrope_config.mrope_position_deltas"
                                     ])
-                            for beam in range(beam_width):
-                                mrope_position_ids.append(
-                                    gen_mrope_position_ids)
-                                multimodal_params_list.append(multimodal_params)
+
+                    if gen_mrope_position_ids is None:
+                        gen_mrope_position_ids = get_default_mrope_position_ids(
+                            [position_id])
+
+                    for beam in range(beam_width):
```

Ruff (0.15.6)

[warning] Loop control variable `beam` not used within loop body

(B007)